src/App.js

```diff
@@ -22,9 +24,6 @@ const fabBtn = {
 
 const App = () => {
   const [moviesList, setMoviesList] = React.useState([]);
-  const [theme, setTheme] = React.useState(
```

Comment:
you removed the theme ?

Reply:
Nope. Moved to diff component.
We need to do the same refactor in other components too.. May be I will create an issue in the repo